packages/fxa-settings/src/models/verification/verification-info.ts

```diff
@@ -18,19 +18,19 @@ const { isEmail, isRequired, isVerificationCode, isHex, isString, isBoolean } =
 const { snakeCase } = KeyTransforms;
 
 export class VerificationInfo extends ModelDataProvider {
-  @bind([isEmail, isRequired])
+  @bind([isRequired, isEmail])
```

Comment:
ok n00b question is `isRequired` still necessary if the value must be `isEmail`?

Reply:
I think exactly how we do this is still up for debate. I think this is actually a really good question, and there's still some work to do on validation in general.

The current reason for both isRequired and isEmail is that a situation could arise where you might allow an email field to be undefined. Therefore, the isEmail validator only runs the check if the string is in fact defined. The checks are run in order, so checking isRequired upfront makes more sense, and is necessary if the isEmail validator allows an undefined value to pass.